tests/_test-utils.js

```diff
@@ -0,0 +1,5 @@
+const execa = require('execa');
+
+module.exports = {
+	runInstrumentedPlop: (...args) => execa('npx', ['nyc', '--silent', 'node', './instrumented/bin/plop.js', ...args]),
```

Comment:
This command will execute the instrumented plop CLI in a subprocess with nyc pulling out the coverage